elasticsearch/esql/esql.py

```diff
@@ -97,7 +97,10 @@ class ESQLBase(ABC):
     commands, used to build ES|QL queries.
     """
 
-    def __init__(self, parent: Optional["ESQLBase"] = None):
+    def __init__(
+        self,
+        parent: Optional["ESQLBase"] = None,
+    ):
```

Comment:
nit: no-op change

Reply:
I blame black for this. I'll revert it and reformat to see if black decides to keep it.